src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -2692,6 +2692,28 @@ public void sendEquipmentChange(@NotNull LivingEntity entity, @NotNull Equipment
 		// Pretend the packet gets sent.
 	}
 
+	@Override
+	public boolean isOp()
+	{
+		return MockBukkit.getMock().getPlayerList().getOperators().stream()
+				.anyMatch(op -> op.getPlayer() == this);
```

Comment:
Can't this use `#contains`?

Reply:
No, since Player doesn't implement OfflinePlayer I have to use this way.